VTEX - GiftCard Hub API.json

```diff
@@ -407,28 +407,20 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "OK",
-                        "content": {
-                            "application/json": {
-                                "schema": {
-                                    "type": "object"
-                                }
-                            }
-                        }
+                        "description": "OK. This endpoint does not return any data in the response body."
```

Comment:
```suggestion
                        "description": "OK"
```

Reply:
Response descriptions should only be the status code name according to our [template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc).

Reply:
We had previously discussed this message for responses where it is necessary to make it clear to the customer that 200-OK means success and there is no content to be displayed.

We should have updated the API template at that time.

Thread: https://vtex.slack.com/archives/C05G6ANSU67/p1701085906819989?thread_ts=1700245355.794799&cid=C05G6ANSU67